app/controllers/students_controller.rb

```diff
@@ -38,7 +38,7 @@ def create
   end
 
   def student_params
-    params.require( :student).permit( :name)
+    params.require( :student).permit( :name, :session_id, :course_id, :attendance_id )
```

Comment:
Do these extra attributes need to be here? When creating a student, I think the only field is `name`.